From 7bb2a54a55f01edc4c55caf8f0da35b384e3b8d7 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Thu, 28 Aug 2014 17:01:34 +0800
Subject: [PATCH 0861/5242] MLK-12284-2 doc: syscon: add clocks as optional
 property

commit  ce24908a49c90518a80f47cca9e3ad5f8ebdbaed from
https://source.codeaurora.org/external/imx/linux-imx.git

User can specify clocks in devicetree which is used for accessing the registers
in this regmap.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
(cherry picked from commit 4a89ef5b579e6fb5640df099ee13939ca6d3a325)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 Documentation/devicetree/bindings/mfd/syscon.txt |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/mfd/syscon.txt b/Documentation/devicetree/bindings/mfd/syscon.txt
index 25d9e9c..6ec0740 100644
--- a/Documentation/devicetree/bindings/mfd/syscon.txt
+++ b/Documentation/devicetree/bindings/mfd/syscon.txt
@@ -13,9 +13,10 @@ Required properties:
 - compatible: Should contain "syscon".
 - reg: the register region can be accessed from syscon
 
-Optional property:
+Optional properties:
 - reg-io-width: the size (in bytes) of the IO accesses that should be
   performed on the device.
+- clocks: clock used for accessing the regmap
 - hwlocks: reference to a phandle of a hardware spinlock provider node.
 
 Examples:
-- 
1.7.9.5

